pkg/virt-api/webhooks/fuzz/fuzz_test.go

```diff
@@ -37,6 +38,14 @@ type testCase struct {
 
 // FuzzAdmitter tests the Validation webhook execution logic with random input: It does schema validation (syntactic check), followed by executing the domain specific validation logic (semantic checks).
 func FuzzAdmitter(f *testing.F) {
+	// Skip on s390x, since it uses go-test
+	// Fuzz testing has it's own test lane and is not run when invoking via bazel-test.
+	// There is no good way to skip the test when using go test directly, as using tags does not work well with bazel.
+	// TODO: Remove when switching to bazel-test
+	if goruntime.GOARCH == "s390x" {
+		f.Skip("Skip fuzz testing when invoking via go-test")
```

Comment:
This is not accurate. What is the problem with fuzz + s390x? Maybe we want to introduce a env on which we skip or we just simply adjust the target to skip fuzz tests?

Reply:
The problem starts with the fact that s390x is not running `make bazel-test` but rather `make go-test`. This means that it does invoke the race detector when running the tests, causing the fuzz test to timeout, since it adds overhead. Now this could be easily fixed by increasing the timeout. However the tests still end up failing, since some of the random input seems to not be allowed by the api validation. In fact, it should fail on x86 as well.
But that brings us to the last point, i don't think this test is run on any other arch currently. At least i could not get it to run when using `make bazel-test`. After talking to @brianmcarey, fuzz testing is supposed to be it's own test lane and not run together with the unit-tests. That is why i choose to disable it for s390x.

Though if you know of a better way to disable it during normal unit-tests, that would be great. As i wrote i only choose to do it this way because i haven't found one that works for both bazel and normal go.

Reply:
As you don't run bazel I would just add `-skip` (see below) for the the `s390x`
```
 -skip regexp
            Run only those tests, examples, fuzz tests, and benchmarks that
            do not match the regular expression. Like for -run and -bench,
            for tests and benchmarks, the regular expression is split by unbracketed
            slash (/) characters into a sequence of regular expressions, and each
            part of a test's identifier must match the corresponding element in
            the sequence, if any.
```

Reply:
I guess `-skip` should work as well, since we only have the 1 fuzz test.

Reply:
Let's change it so we can still look at the fuzzing, why it is failing and so on...

Reply:
Agreed, i checked today and using `-skip` in the script works. I have added it to the `hack/build-go.sh` along with a comment.